Sources/Apodini/Semantic Model Builder/InterfaceExporter/InterfaceExporter.swift

```diff
@@ -85,6 +90,13 @@ protocol InterfaceExporter: BaseInterfaceExporter {
     func retrieveParameter<Type: Decodable>(_ parameter: EndpointParameter<Type>, for request: ExporterRequest) throws -> Type??
 }
 
+extension BaseInterfaceExporter {
+    static var parameterNamespace: [ParameterNamespace] {
+        [] // default is to return nothing, to distinguish if exporter did actually set something
```

Comment:
As far as I can see the empty array is replaced with a `[.global]` version in **ParameterNamespace.swift** in `nameCollisionCheck` and we don't check for an empty array anywhere else. It might be more clearer for someone who checks out `parameterNamespace` to find the default option that will eventually be used here instead in `nameCollisionCheck`.